lib/Guzzle/AuthenticatorSubscriber.php

```diff
@@ -88,8 +88,16 @@ public function loginIfRequested(CompleteEvent $event)
             return;
         }
 
+        $body = $event->getResponse()->getBody()->getContents();
+
+        if ('' === $body) {
```

Comment:
`$event->getResponse()` and `getBody()` both can return `null`, maybe check that too?

Reply:
@tcitworld I've updated the PR to check if `getBody()` is null. I think we can assume that `getResponse()` is not nullable